src/mpid/ch4/shm/posix/eager/iqueue/iqueue_init.c

```diff
@@ -64,8 +64,8 @@ int MPIDI_POSIX_iqueue_init(int rank, int size)
 
     transport->my_terminal = &transport->terminals[MPIDI_POSIX_global.my_local_rank];
 
-    mpi_errno = MPIDU_genq_shmem_queue_init(transport->my_terminal, transport->cell_pool,
-                                            MPIDU_GENQ_SHMEM_QUEUE_TYPE__MPSC);
+    mpi_errno = MPIDU_genq_shmem_queue_init(transport->my_terminal,
+                                            MPIDU_GENQ_SHMEM_QUEUE_TYPE__NEM);
```

Comment:
Naming of queue type should be general. Not referring to NEM. In fact, I think it is better to just delete inversed queue. Then this can simply be `MPSC` queue type.

Reply:
done.